pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -79,86 +77,101 @@ In plain language, this code says:
 
   The USWDS source code is the core of the design system. It contains all the styles for USWDS components as well as the design language of Sass tokens and functions used to build those components. USWDS source code has its own Sass entry point, which lives in the `node_modules` directory when you install USWDS with npm.
 
-  The entry point itself is called `uswds.scss`, and it’s found in the `/dist/scss` directory of the USWDS npm package. When you install with npm, the complete path looks something is typically `./node_modules/uswds/dist/scss/uswds.scss`.
+The entry point itself is called `uswds.scss`, and it’s found in the `/dist/scss` directory of the USWDS npm package. When you install with npm, the complete path looks something is typically `./node_modules/uswds/dist/scss/uswds.scss`.
 
 - **Build new work on top of that foundation**: Finally, add any custom project styles built from design system code.
 
-  After you import the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
+After you import the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
 
-## Step 2: Install uswds-gulp
-From your project’s root, run the following command to install `uswds-gulp` and all its dependencies:
+## Step 2: Install uswds-compile
+From your project’s root, run the following command to install `uswds-compile` and all its dependencies:
```

Comment:
This _could_ be simplified to say install `uswds-compile`.

Reply:
Agreed and implemented—thanks, James!